bin/setup

```diff
@@ -47,14 +39,7 @@ host    all             all             127.0.0.1/32            trust
 host    all             all             ::1/128                 trust
 host    all             all             0.0.0.0/0               password 
 EOF
-sudo service postgresql restart
-
-# Need to upgrade pip 
-pip install --upgrade setuptools
 
-# Setup Python Requirements
-cd ~/accounting
-pip install -r requirements.txt
-
-# Initial Migration
+# Begin DB services with migrations finished.
```

Comment:
@UmanShahzad This is a little confusing: The code below restarts `postgresql` before running migrations?

Reply:
@itsjeyd Yeah that comment is pretty bad. I think I should just delete the comment, the commands are very self-descriptive.

Reply:
@UmanShahzad Nice :)